api/app/GraphQL/Queries/CountPoolCandidatesByPool.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\GraphQL\Queries;
+
+use App\Models\PoolCandidate;
+use App\Models\User;
+use Database\Helpers\ApiEnums;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Facades\Log;
+use App\Models\Pool;
+
+final class CountPoolCandidatesByPool
+{
+    /**
+     * @param  null  $_
+     * @param  array{}  $args
+     */
+    public function __invoke($_, array $args)
+    {
+        $filters = $args["where"];
+
+        // query counts pool candidate rows, so start on that model
+        $queryBuilder = PoolCandidate::query();
+
+        // pool candidate filters go here
+
+        // candidate pool scope
+        if (array_key_exists('pools', $filters)) {
+            // pool candidate filter uses Pool while Applicant filter users IdInput
+            $pools = array_map(function ($id) {
+                return ['id' => $id];
+            }, $filters['pools']);
+            PoolCandidate::filterByPools($queryBuilder, $pools);
+        }
+
+        $queryBuilder->whereHas('user', function (Builder $userQuery) use ($filters) {
+            // user filters go here
+
+            // user status scope
+            User::scopeAvailableForOpportunities($userQuery);
+
+            // has diploma
```

Comment:
```suggestion
            // hasDiploma
```

Reply:
Thanks!  Fixed in [Tidy up based on reviews](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4455/commits/74587e85db9aeb7ae9cd36053f21c2d005a4081a).